test/spec/geo/leaflet/leaflet-cartodb-layer-group-view.spec.js

```diff
@@ -1,309 +1,55 @@
 /* global L */
-var CartoDBLayer = require('../../../../src/geo/map/cartodb-layer.js');
-var LayersCollection = require('../../../../src/geo/map/layers.js');
-var CartoDBLayerGroup = require('../../../../src/geo/cartodb-layer-group.js');
+var cartoLayerGroupViewTests = require('../shared-tests-for-carto-layer-group');
 var LeafletCartoDBLayerGroupView = require('../../../../src/geo/leaflet/leaflet-cartodb-layer-group-view');
 
 describe('leaflet-cartodb-layer-group-view', function () {
-  var container;
-  var leafletMap;
-
-  var cartoDbLayer0;
-  var cartoDbLayer1;
-  var layerGroupView;
-
-  var layerGroupModelMock;
-  var layersCollection;
-
-  var engineMock = {
-    on: jasmine.createSpy('on')
-  };
-
-  var mapModelMock = {
-    on: jasmine.createSpy('on'),
-    isFeatureInteractivityEnabled: jasmine.createSpy('isFeatureInteractivityEnabled').and.returnValue(false)
-  };
-
-  beforeEach(function () {
+  /**
+   * Helper function used to get a google map and a container in the shared tests.
+   */
+  function setUp () {
```

Comment:
Perhaps this could be `createNativeMap(container)` so that the container is managed by `shared-tests-for-carto-layer-group.js`.

Reply:
Done